coverage_config_aarch64.json

```diff
@@ -1,5 +1,5 @@
 {
-  "coverage_score": 84.8,
+  "coverage_score": 82.4,
```

Comment:
The big difference between aarch64 and x86_64 is very weird since the code is not at all different.

Reply:
Also interesting is, after updating to new CI, X86 coverage dropped 0.1%.

Reply:
That is probably because of the Rust version update.

Reply:
I compared the report generated on x86 and aarch64. It looks like there are some mistakes in Kcov report for the latter one.

Reply:
Is this possible?
![](https://github.com/michael2012z/vm-memory/blob/tmp/kcov_.PNG?raw=true)

Reply:
Yeah, I was expecting that because in other crates we also get 1% difference at least even though the same code applies to both of them.